components/user/activity/constants.ts

```diff
@@ -0,0 +1,9 @@
+export enum ACTIVITY_TYPES {
```

Comment:
this values are being setted from /lib/api/hooks, it would be great that this constants were being declared somewhere in /lib so both the activity component and the API hooks use the same constants.